pages/index.js

```diff
@@ -1,9 +1,79 @@
-import Feed from "./components/feed";
+import Head from "next/head";
+import Image from "next/image";
+import Link from "next/link";
+import { Inter } from "next/font/google";
+import styles from "@/styles/Home.module.css";
```

Comment:
We initially agreed on styling the capstone app with only styled components. Mixing css modules and styled components is considered bad practice and unfortunately I have to ask you to rewrite your styling into styled components.